.changeset/clean-beds-laugh.md

```diff
@@ -0,0 +1,6 @@
+---
+'@compiled/eslint-plugin': minor
+'@compiled/react': minor
+---
```

Comment:
```suggestion
'@compiled/eslint-plugin': patch
---
```

Reply:
no need for changeset for `@compiled/react` cuz the customer's code doesn't change